src/main/resources/db/changelog/source/20200426.REL1-22.create-source-table.yml

```diff
@@ -0,0 +1,24 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
add - logicalFilePath: no-path [https://github.com/feederr/feeder-authorization/blob/3e7505a5357ddf1461fc535cfe809ec032e82502/src/main/resources/db/changelog/user/20200419.REL1-11.add-unique-constraint-on-username.yaml#L3](like it done there)